### PR TITLE
closes #29 review api

### LINKED ISSUE #29: Create Review API

### PR DESCRIPTION
**Pull request by gabrielwr**
closes #29

### CODE REVIEW

File: server/review.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    mustBeLoggedIn,
+    (req, res, next) =>
+    Review.findById(req.params.id)
+    .then(review => review.destroy())
+    .then(wasDestroyedBool => {
+      if (wasDestroyedBool) {
+        res.sendStatus(204)
+      } else {
+        const err = Error('campus not destroyed')
+        err.status = 400
```
Comment (lada-tech) [inline server/review.js]: the error message should be different )

File: server/api.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ api
   .get('/heartbeat', (req, res) => res.send({ok: true}))
   .use('/auth', require('./auth'))
   .use('/users', require('./users'))
+  .use('/reviews', require('./review'))
```
Comment (JeffKandel) [inline server/api.js]: please change the file name to reviews to stay in line with previous things like users.

**Review by lada-tech — APPROVED** [review]
fix error in delete

File: server/review.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  // TO DO: make sure that this review belongs to user and review
+    //must be logged in to edit?
+    mustBeLoggedIn,
+    (req, res, next) =>
+    Review.findById(req.params.id)
+    .then(review => review.update(req.body))
+    .then(updatedreview => res.json(updatedreview))
+    .catch(next))
+
+    .delete('/:id',
```
Comment (JeffKandel) [inline server/review.js]: Move this back two spaces and delete line 41 or there will be an error

**Review by JeffKandel — CHANGES_REQUESTED** [review]
See below, minor cosmetic changes, but one will throw an error!

**Review by JeffKandel — APPROVED** [review]
There is room to build out more functionality, but this is a good start.  Let's merge and keep moving forward.

**Pull request merged**